src/jdk.jartool/share/man/jar.md

```diff
@@ -106,6 +106,10 @@ argument is the first argument specified on the command line.
 `-d` or `--describe-module`
```

Comment:
I'm going to start with some trivial nits and other review comments. The copyright year on this file will need a update.